grafana/dashboards/storageGrid.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: 1
+
+providers:
+  - name: 'Harvest cDOT'
```

Comment:
Harvest StorageGrid?

Reply:
Doesn't seem like this name matters? With the changes above, the dashboards are imported like so: 
<img width="268" alt="image" src="https://user-images.githubusercontent.com/242252/208122343-5d4b63b1-6793-471d-b567-dfea395af903.png">